src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -187,6 +190,9 @@ export class EmbeddablePanel extends React.Component<Props, State> {
     if (this.embeddableRoot.current) {
       this.props.embeddable.render(this.embeddableRoot.current);
     }
+    this.props.embeddable.actionStorage
+      .count()
+      .then(drilldownCount => this.setState({ drilldownCount }));
```

Comment:
Here we should check if component is still mounted:

```suggestion
      .then(drilldownCount => if(this.mounted) this.setState({ drilldownCount }));
```